3style-text/styles.css

```diff
@@ -0,0 +1,89 @@
+/* ---font styles--- */
+
+body {
+  font-family: "Public Sans", Verdana, sans-serif;
+  font-size: 18px;
+}
+
+
+h1, h2, h3, h4, h5, h6 {
```

Comment:
Great work combining styles here rather than declaring this each time for each heading!